dns/bind/src/opnsense/mvc/app/models/OPNsense/Bind/General.xml

```diff
@@ -115,6 +115,16 @@
             <Multiple>N</Multiple>
             <Required>Y</Required>
         </dnssecvalidation>
+        <syslog type="OptionField">
+            <Required>N</Required>
+            <Multiple>Y</Multiple>
+            <OptionValues>
```

Comment:
There doesn’t seem to be a way to turn this off again even though it’s not “required”. Empty implied multi select value seems a little too dense,  it might just be me.

Reply:
dnsbl.xml uses the same method for multiple-select fields; I took this as precedent and replicated. I used the [official documentation](https://docs.opnsense.org/development/frontend/models_fieldtypes.html#optionfield) as a reference. As far as I can see, "de-selection" is the equivalent of disabled.

The [same documentation](https://docs.opnsense.org/development/frontend/models_fieldtypes.html#optionfield) does not mention a means to unset the option.